tests/Tree/CollectionTest.php

```diff
@@ -38,4 +38,38 @@ public function testToTreeWithEmptyCollection()
 
         $this->assertEmpty($tree);
     }
+
+    public function testLoadTreePathRelations()
+    {
+        $limit = User::count();
+
+        $loaded = 0;
+
+        User::retrieved(function () use (&$count) {
+            $count++;
```

Comment:
`$loaded` is created to store this count in, but it's never updated. `$count` is unused. Fixed this in my proposal below